packages/gatsby-utils/src/cpu-core-count.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @type {import('../index').cpuCoreCount}
+ */
+const cpuCoreCount = (useEnvVar = true) => {
```

Comment:
using ignoreEnvVar might make more sense than setting it to true by default
```suggestion
const cpuCoreCount = (ignoreEnvVar) => {
```